README.md

```diff
@@ -226,9 +230,82 @@ Hey that seemed to work! Here is the updated [elm.json](elm.json):
 }
 ```
 
-And now we would want to try importing the types into our Main.elm
-**under development**
+And now we would want to try importing the types into our Main.elm.
+Per code as example help from [here](https://github.com/vsoch/elm-app-catalog/pull/1/files)
+I was directed to look at [documentation](https://package.elm-lang.org/packages/mdgriffith/elm-ui/latest/Element) to familiarize myself with the Element namespace. This is 
+akin to looking up functions and usage for any module. Things that I experimented
+with include:
+
+ - changing a variable named "Model" to "Counter" because it represents an integer for the counter number, and I didn't think "model" described that accurately.
```

Comment:
Note: Both `Model` and `Msg` are types, not variables. 

There are no variables in Elm. It really takes immutability seriously. You can only define constants, and only once in a namespace.

Example:

```elm
update : Action -> Counter -> Counter
update action counter =
    let
        value1 =
            "Some string"

        value2 =
            "Some other string"

        value1 =
            "Changing value 1"
    in
    case action of
        -- snip
```

This will give you a compiler error:

```
elm make src/Main.elm 
Detected problems in 1 module.
-- NAME CLASH ----------------------------------------------------- src/Main.elm

This `let` expression defines `value1` more than once! One here:

77|         value1 =
            ^^^^^^
And another one here:

71|         value1 =
            ^^^^^^
How can I know which one you want? Rename one of them!
```